Taejin -

TaeJinKim0930 opened the pull request.

TaeJinKim0930 committed:
Stock DTO added

재고, 재고실사, 품목별 재고, 창고별 재고 DTO 파일들 작업 및 주석 처리 완료

TaeJinKim0930 committed:
테스트 케이스 + pull 위해서 추가

hinata7010 reviewed (changes requested):
Test 보완 필요 log를 찍어보거나 assertThat과 같은 메소드로 검증하는 과정이 필요합니다.

TaeJinKim0930 committed:
Test: StockMapper.xml Query + mapper param + test input 값 수정

Fix(StockMapper.xml): param 값 categoryId -> main/middle/subCategory 로 각각 수정
Fix(StockMapper.java): 위와 동일
Test(StockMApperTests): log.info 값 추가
